common-npm-packages/AzureRmDeploy-common/azure-arm-rest/azure-arm-app-service.ts

```diff
@@ -263,6 +263,13 @@ export class AzureAppService {
                 tl.debug(`Slot setting updated for key : ${addProperties[key].name}`);
                 isNewValueUpdated = true;
             }
+            else if (addProperties[key].slotSetting == false || (addProperties[key].slotSetting == null)) {
```

Comment:
Update pull request description and add issue link. it will be easier to understand which issue you are trying to fix